snap/hooks/configure

```diff
@@ -45,4 +45,6 @@ then
     touch ${SNAP_COMMON}/initial-migrations-run
 fi
 
+mdkir "${SNAP_DATA}/media"
```

Comment:
The `configure` hook gets run each time the snap is upgraded; the name might be a bit misleading. This would fail the first time the snap is upgraded, as `mkdir` will fail to create an existing directory.

It's also *slightly* misspelled :wink: